/home/z5102138/anaconda3/envs/py36/bin/python
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
im_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_1
./test_yeast5/result_MLP_both2_200_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
im_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_2
./test_yeast5/result_MLP_both2_200_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
im_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_3
./test_yeast5/result_MLP_both2_200_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
im_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_4
./test_yeast5/result_MLP_both2_200_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
im_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_5
./test_yeast5/result_MLP_both2_200_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
im_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_1
./test_yeast5/result_MLP_both2_500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
im_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_2
./test_yeast5/result_MLP_both2_500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
im_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_3
./test_yeast5/result_MLP_both2_500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
im_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_4
./test_yeast5/result_MLP_both2_500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
im_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_5
./test_yeast5/result_MLP_both2_500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
im_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_1
./test_yeast5/result_MLP_both2_1000_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
im_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_2
./test_yeast5/result_MLP_both2_1000_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
im_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_3
./test_yeast5/result_MLP_both2_1000_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
im_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_4
./test_yeast5/result_MLP_both2_1000_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
im_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_5
./test_yeast5/result_MLP_both2_1000_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
im_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_1
./test_yeast5/result_MLP_both2_1500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
im_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_2
./test_yeast5/result_MLP_both2_1500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
im_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_3
./test_yeast5/result_MLP_both2_1500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
im_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_4
./test_yeast5/result_MLP_both2_1500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
im_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_5
./test_yeast5/result_MLP_both2_1500_im_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
both_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_1
./test_yeast5/result_MLP_both2_200_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
both_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_2
./test_yeast5/result_MLP_both2_200_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
both_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_3
./test_yeast5/result_MLP_both2_200_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
both_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_4
./test_yeast5/result_MLP_both2_200_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_200
both_num_10_half
./test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_5
./test_yeast5/result_MLP_both2_200_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_200/record_1/MLP_both2_200_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
both_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_1
./test_yeast5/result_MLP_both2_500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
both_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_2
./test_yeast5/result_MLP_both2_500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
both_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_3
./test_yeast5/result_MLP_both2_500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
both_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_4
./test_yeast5/result_MLP_both2_500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_500
both_num_10_half
./test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_5
./test_yeast5/result_MLP_both2_500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_500/record_1/MLP_both2_500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
both_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_1
./test_yeast5/result_MLP_both2_1000_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
both_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_2
./test_yeast5/result_MLP_both2_1000_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
both_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_3
./test_yeast5/result_MLP_both2_1000_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
both_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_4
./test_yeast5/result_MLP_both2_1000_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1000
both_num_10_half
./test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_5
./test_yeast5/result_MLP_both2_1000_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1000/record_1/MLP_both2_1000_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
both_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_1
./test_yeast5/result_MLP_both2_1500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
both_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_2
./test_yeast5/result_MLP_both2_1500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
both_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_3
./test_yeast5/result_MLP_both2_1500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
both_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_4
./test_yeast5/result_MLP_both2_1500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both2_1500
both_num_10_half
./test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_5
./test_yeast5/result_MLP_both2_1500_both_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both2_1500/record_1/MLP_both2_1500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
normal_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_1
./test_yeast5/result_MLP_both3_200_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
normal_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_2
./test_yeast5/result_MLP_both3_200_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
normal_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_3
./test_yeast5/result_MLP_both3_200_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
normal_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_4
./test_yeast5/result_MLP_both3_200_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
normal_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_5
./test_yeast5/result_MLP_both3_200_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_500
normal_num_10_half
./test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_1
./test_yeast5/result_MLP_both3_500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_500
normal_num_10_half
./test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_2
./test_yeast5/result_MLP_both3_500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_500
normal_num_10_half
./test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_3
./test_yeast5/result_MLP_both3_500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_500
normal_num_10_half
./test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_4
./test_yeast5/result_MLP_both3_500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_500
normal_num_10_half
./test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_5
./test_yeast5/result_MLP_both3_500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_500/record_1/MLP_both3_500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1000
normal_num_10_half
./test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_1
./test_yeast5/result_MLP_both3_1000_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1000
normal_num_10_half
./test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_2
./test_yeast5/result_MLP_both3_1000_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1000
normal_num_10_half
./test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_3
./test_yeast5/result_MLP_both3_1000_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1000
normal_num_10_half
./test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_4
./test_yeast5/result_MLP_both3_1000_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1000
normal_num_10_half
./test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_5
./test_yeast5/result_MLP_both3_1000_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1000/record_1/MLP_both3_1000_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1500
normal_num_10_half
./test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_1
./test_yeast5/result_MLP_both3_1500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_1'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1500
normal_num_10_half
./test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_2
./test_yeast5/result_MLP_both3_1500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_2'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1500
normal_num_10_half
./test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_3
./test_yeast5/result_MLP_both3_1500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_3'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1500
normal_num_10_half
./test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_4
./test_yeast5/result_MLP_both3_1500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_4'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_1500
normal_num_10_half
./test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_5
./test_yeast5/result_MLP_both3_1500_normal_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 301, in <module>
    net = torch.load(model_name)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './test_yeast5/model_MLP_both3_1500/record_1/MLP_both3_1500_5'
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
bm_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_1
./test_yeast5/result_MLP_both3_200_bm_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
bm_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_2
./test_yeast5/result_MLP_both3_200_bm_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
bm_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_3
./test_yeast5/result_MLP_both3_200_bm_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
bm_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_4
./test_yeast5/result_MLP_both3_200_bm_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/srv/scratch/z5102138/cifar10/all_train_data.pkl
/srv/scratch/z5102138/cifar10/all_test_data.pkl
MLP_both3_200
bm_num_10_half
./test_yeast5/model_MLP_both3_200/record_1/MLP_both3_200_5
./test_yeast5/result_MLP_both3_200_bm_num_10_half/record_1/
----------------------



Traceback (most recent call last):
  File "./classifier_MLP/test.py", line 345, in <module>
    precision = skmet.precision_score(y_true=true_label, y_pred=predict_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1672, in precision_score
    zero_division=zero_division)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1484, in precision_recall_fscore_support
    pos_label)
  File "/home/z5102138/anaconda3/envs/py36/lib/python3.6/site-packages/sklearn/metrics/_classification.py", line 1316, in _check_set_wise_labels
    % (y_type, average_options))
ValueError: Target is multiclass but average='binary'. Please choose another average setting, one of [None, 'micro', 'macro', 'weighted'].
